sklearn/tests/test_pipeline.py

```diff
@@ -519,7 +519,8 @@ def test_make_union_kwargs():
     # invalid keyword parameters should raise an error message
     assert_raise_message(
         TypeError,
-        'Unknown keyword arguments: "transformer_weights"',
+        "make_union() got an unexpected \
```

Comment:
This \ is where your code is failing. It means that the expected error message must contain eight spaces! String literals can be joined just by having them one after the other with only whitespace in between

Reply:
Thank you for the input. I just pushed a commit resolving that issue. I think it works, but let me know if there's anything else I need to do!